websites/Y/YTPLR/presence.ts

```diff
@@ -0,0 +1,82 @@
+const presence = new Presence({
+		clientId: "1284161421957136486",
+	}),
+	strings = presence.getStrings({
+		play: "Watching",
+		pause: "Paused",
+	}),
```

Comment:
See the [localization repository](https://github.com/PreMiD/Localization/blob/main/src/Presence/general.json) for keys to use for strings.